### ISSUE TITLE
Demo movement code does not work

### ISSUE THREAD

**Issue by shisha101** [shisha101/ROSRagnarEDU#4]
After connecting the robot using

```
roslaunch ragnar_support bringup.launch robot_ip:=10.0.1.39
```

then using the command

```
rosrun ragnar_kinematics ragnar_demo_motions 
```

the following output is observed showing that the points have been sent to the Ragnar but **no** movement is observed neither in Rviz nor in reality. the joint_states topic does not change either, this might be linked to #3 

```
[ INFO] [1455896587.281673154]: Receiving joint trajectory message
[ INFO] [1455896587.283135351]: Loading trajectory, setting state to streaming
[ INFO] [1455896587.435665644]: Executing trajectory of size: 106
[ INFO] [1455896587.441173656]: Point[0 of 106] sent to controller
...
[ INFO] [1455896588.014771081]: Point[105 of 106] sent to controller
[ INFO] [1455896588.019941594]: Trajectory streaming complete, setting state to IDLE
```

**Comment by shisha101** [comment]
@ipa-mirb

**Comment by shisha101** [comment]
- The **/joint_path_command** topic is receiving the correct values
- The **/joint_trajectory_action/feedback** topic is receiving **nothing**
- The **/feedback_states** topic is receiving **nothing**
- The **/joint_trajectory_action/goal**  topic is receiving **nothing**
- The **/joint_states** topic is receiving **no updates and stays static**

**Comment by shisha101** [comment]
The reason for this was that the encoders were not outputting the correct values, which has bee temporarily fixed by using the web-interface to enable the encoders by clicking on the **home** button

this is being addressed in issue #5 

The rest of the topics are not active, advertised but not being used at the moment this will be addressed in a separate issue
